ArduPlane/ArduPlane.cpp

```diff
@@ -849,6 +849,28 @@ bool Plane::set_land_descent_rate(float descent_rate)
     return false;
 }
 
+// returns true if vehicle is landing.
+bool Plane::is_landing() const
+{
+#if HAL_QUADPLANE_ENABLED
+    if (plane.quadplane.in_vtol_land_descent()) {
```

Comment:
Why `in_vtol_land_descent` and not `in_vtol_land_final` ? I'm not really sure which is the correct answer. Potentially land descent could start at a 100m altitude or or more.

Reply:
I'd say either one and/or both are valid landing states. If I was a landing gear or an LED blinky status thing I'd want to know "landing" meant a land descent and not just the final moments